test/functional/test_framework/test_node.py

```diff
@@ -91,6 +91,12 @@ def wait_for_rpc_connection(self):
             attempts -= 1
         raise AssertionError("Unable to connect to bitcoind")
 
+    def get_wallet_rpc(self, wallet_name):
+        assert self.rpc_connected
+        assert self.rpc
+        wallet_path = "wallet/%s" % wallet_name
+        return self.rpc / wallet_path
```

Comment:
By magic :)

Take a look at https://github.com/bitcoin/bitcoin/blob/e526ca6284b9e13be1b912b80dd73a34e739b539/test/functional/test_framework/authproxy.py#L195 and https://github.com/bitcoin/bitcoin/blob/e526ca6284b9e13be1b912b80dd73a34e739b539/test/functional/test_framework/coverage.py#L59

The interface here is supposed to be similar to Python's [pathlib](https://docs.python.org/3/library/pathlib.html)

This functionality was added to support the multiwallet endpoints in https://github.com/bitcoin/bitcoin/commit/979d0b8a6533de58b6352666d90fff33841db63d . Take a look at the `multiwallet.py` test to see how it's currently used.